src/player.js

```diff
@@ -1949,6 +1969,9 @@ export default class Player extends FakeEventTarget {
       this._engine
         .load(startTime)
         .then(data => {
+          if (this.isLive()) {
```

Comment:
After load with explicit startTime you'll not on live edge

Reply:
It is "safe" to assume that if someone set start time for live he will want to be in DVR and not live.
The default behaviour is to be on lice edge when loading a live stream without a start time.

Reply:
Live doesn't include DVR? I mean isLive will return true for DVR , right?

Reply:
Live will return true for live with and without DVR.

Reply:
OK, the whole fix is about DVR cause seekbar doesn't exist for live, so if you start with start time you're not in the live edge but set liveEdge to true

Reply:
was there a question there? :-)

Reply:
Nop, just explanation for the problematic scenario :)